DEVELOPMENT.md

```diff
@@ -110,10 +110,10 @@ Tekton Dashboard does not involve any custom resource definitions, we only inter
 You can stand up a version of the dashboard on-cluster (to your
 `kubectl config current-context`):
 
-First install and build the npm project.
+First install and build the npm project. Install with a clean slate of dependencies using npm ci. 
```

Comment:
'Install with a clean slate of dependencies, an existing node_modules folder in the project root will be removed.'